src/ToDo/Business/ToDoTask.cs

```diff
@@ -39,11 +39,11 @@ internal ToDoTask(string listId, TaskData data)
 
 	public string Id { get; } // No public init: this can be set only from a data
 
-	public string? Importance { get; init; }
+	public string? Importance { get; set; }
 
 	public bool IsReminderOn { get; init; }
 
-	public string? Status { get; init; }
+	public string? Status { get; set; }
```

Comment:
We want to keep record immutable, use the `with` syntax instead: `var updatedTask = task with { Importance = "important", Status = "completed" };`